pandas/core/groupby/ops.py

```diff
@@ -148,7 +148,7 @@ def _get_group_keys(self):
             # provide "flattened" iterator for multi-group setting
             return get_flattened_iterator(comp_ids, ngroups, self.levels, self.codes)
 
-    def apply(self, f, data: FrameOrSeries, axis: int = 0):
+    def apply(self, f: Callable, data: FrameOrSeries, axis: int = 0):
```

Comment:
Is it possible to subscript Callable?

Reply:
I _think_ this can be `Callable[[FrameOrSeries], Any]` (not sure how much more specific `Any` could be) and it seems to pass `mypy`